src/lib/block_if.c

```diff
@@ -101,6 +110,8 @@ struct blockif_ctxt {
 	off_t			bc_size;
 	int			bc_sectsz;
 	int			bc_psectsz;
+	int			bc_delete_alignment; /* f_bsize always a power of 2 */
```

Comment:
If you make this a `size_t` then I think a bunch of the explicit casting can go away.